backend/app/routers/metadata_datasets.py

```diff
@@ -26,6 +27,7 @@
     patch_metadata,
     MetadataDelete,
 )
+from app.search.connect import insert_record, update_record, delete_document_by_id
```

Comment:
I got an error here about the `update_record` function

```
    await self.middleware_stack(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 75, in __call__
    raise exc
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 64, in __call__
    await self.app(scope, receive, sender)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 680, in __call__
    await route.handle(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 275, in handle
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 65, in app
    response = await func(request)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/routing.py", line 231, in app
    raw_response = await run_endpoint_function(
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/routers/metadata_datasets.py", line 251, in update_dataset_metadata
    result = await patch_metadata(md, contents, db)
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/models/metadata.py", line 325, in patch_metadata
    raise e
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/models/metadata.py", line 323, in patch_metadata
    update_record(es, "metadata", doc, metadata["_id"])
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/search/connect.py", line 79, in update_record
    es_client.update(index=index_name, id=id, body=body)
AttributeError: 'Depends' object has no attribute 'update'
```

Reply:
To trigger this error, just try update any of the existing metadata
<img width="458" alt="image" src="https://user-images.githubusercontent.com/13950475/201438666-388dbbe3-0a49-4964-b2e0-b04c837e1ffc.png">

Reply:
Fixed